images/tiler-server/Dockerfile

```diff
@@ -50,4 +50,4 @@ COPY ./start.sh .
 COPY ./expire-watcher.sh .
 COPY ./seed-by-diffs.sh .
 COPY ./tile_cache_downloader.sh .
-CMD ./start.sh & ./tile_cache_downloader.sh & ./expire-watcher.sh
+# CMD ./start.sh & ./tile_cache_downloader.sh & ./expire-watcher.sh
```

Comment:
This shouldn't be commented out right?

Reply:
@geohacker yes, you are right, it should not be applied to oms-seed, for ohm  the tiler server and cache cleaner are running in different  pods, and we need to pass different  the starting commands.